examples/vae_simple.py

```diff
@@ -0,0 +1,322 @@
+import sys
+import os
+import numpy as np
+import theano
+import theano.tensor as T
+import lasagne as nn
+import time
+import pickle
+from PIL import Image
+from lasagne.layers import MergeLayer
+from theano.tensor.shared_randomstreams import RandomStreams
```

Comment:
I replaced it but it did not appear to make a difference in running time. Regardless, I will try to be consistent with rest of Lasagne so glad to leave this one in.